changes/feat-14409.en.md

```diff
@@ -0,0 +1,7 @@
+Add two rule functions to convert UTF-8 strings to UTF-16-little-endian for compatibility with SQL Server versions that do not support UTF-8.
+
+- Convert a UTF-8 string to UTF-16-little-endian.
+  `str_utf16_le/1`
+
+- Convert any string or Binary to SQL Server hex binary format with `0x` prefix.
+  `sqlserver_bin2hexstr/1`
```

Comment:
If it is just used for the SQL Server, why not move this function to the SQL Server Connector or Driver?

Reply:
The binary convert with prefix should be processed by rule engine.